fr/common.json

```
@@ -0,0 +1,452 @@
+      "noRouteIdOfStart": "Itinéraire Privé",
+      "totalDeal": "Dégâts",
+      "equipments": "Équipement",
+      "matchVersion": "Version",
+      "matchId": "ID du Match",
+      "playerKilledMe": "M'a tué",
+      "iKilledPlayer": "Tué par moi",
+      "playerKilledTeammate": "Tué par un allié",
+      "teammateKilledPlayer": "Tué par un allié",
+      "killedWickline": "Wickeline",
+      "killedAlpha": "Alpha",
+      "killedOmega": "Oméga",
+      "deadByWildAnimal": "Tué par un animal",
+      "gaveUp": "Abandonné"
+    }
+  },
+  "database": {
+    "title": "Base de Données",
+    "items": {
+      "title": "Catalogue",
+      "description": "Options d'objets d'Eternal Return, descriptions, statistiques, taux de sélection, taux de victoire",
+      "itemTree": "Recette",
+      "charactersUsing": "Personnages utilisés le plus souvent",
+      "pleaseSelectItem": "Veuillez choisir un article.",
+      "noItemMatchCondition": "Pas d'objet"
+    }
+  },
+  "leaderboard": {
+    "playCountWithUnit": "{{count}}\nParties",
+    "mmr": {
+      "title": "Classement MMR",
+      "description": "Eternal Return solo, duo, escouade, classement MMR, meilleurs joueurs, Immortel MMR",
+      "top1": "TOP 1",
+      "top200": "TOP 200",
+      "top1000": "TOP 1000",
+      "mostPickDistributionRange": "Personnage/Arme des meilleurs joueurs {{start}}~{{end}}"
+    },
+    "artisan": {
+      "title": "Classement d'Artisanat",
+      "description": "Eternal Return Classement d'Artisanat en solo, duo, escouade"
+    }
+  },
+  "doubleSidedVariationChart": {
+    "inactive": "NC",
+    "inactiveMMR": "NC",
+    "event": {
+      "version": "PATCH {{version}}"
+    },
+    "tooltip": {
+      "characterWeapon": "{{characterWeapon}} / {{metrics}}: {{value}}",
+      "mmrFlow": "Dernière MMR: {{last}} (max.: {{max}}, min.: {{min}})"
+    }
+  },
+  "widget": {
+    "playerNotFound": "Joueur Introuvable",
+    "recent3Matches": "3 dernières parties",
+    "nextAutoUpdate": "Mise à jour dans {{time, seconds}}",
+    "updating": "Mise à jour..."
+  },
+  "masteryImprovement": {
+    "average": "Moyenne",
+    "perMinute": "{{number}}/min."
+  },
+  "itemTooltip": {
+    "gather": "Récolte",
+    "monster": "Chasse",
+    "area": "Zone",
+    "supply": "Colis Aérien",
+    "specificCharacterOnly": "Fabricable par {{name}}",
+    "maxStackCount": "Quantité max",
+    "initialCount": "Quantité initiale",
+    "buildsInto": "Transformable en",
+    "bulletCapacity": "Munitions max",
+    "bulletCapacityValue": "{{count}}",
+    "chargeCapacity": "Charges max",
+    "chargeCapacityValue": "{{count}} (recharge : {{chargeTime, seconds}})"
+  },
+  "textTooltip": {
+    "tier": "Le rang représente la puissance et la popularité du personnage, compte tenu du taux de victoire, le pourcentage de classement moyen, les éliminations, les modifications du MMR et le taux de mortalité. Il peut différer même si le Personnage est identique selon les échantillons.",
```

In this tooltip `tier` doesn't mean the rank.
It means... `degré` or something.

```suggestion
    "tier": "Le Tier représente la puissance et la popularité du personnage, compte tenu du taux de victoire, le pourcentage de classement moyen, les éliminations, les modifications du MMR et le taux de mortalité. Il peut différer même si le Personnage est identique selon les échantillons.",
```

---

It got changed accordingly to your suggestion